browser_patches/webkit/patches/bootstrap.diff

```diff
@@ -1680,6 +1680,20 @@ index ddba948c2a08f4a3789773c7ce6c1c1f16d5f11e..c8399e021f1975ada36a7c07de295e1b
  				2D8B92F4203D13E1009C868F /* UnifiedSource519.cpp in Sources */,
  				2D8B92F5203D13E1009C868F /* UnifiedSource520.cpp in Sources */,
  				2D8B92F6203D13E1009C868F /* UnifiedSource521.cpp in Sources */,
+diff --git a/Source/WebCore/dom/UserGestureIndicator.cpp b/Source/WebCore/dom/UserGestureIndicator.cpp
+index dfec93b644f72a51bad0bebf396da61c57f6e428..134b0601ca448da84d46c47483c0e8d650c6148c 100644
+--- a/Source/WebCore/dom/UserGestureIndicator.cpp
++++ b/Source/WebCore/dom/UserGestureIndicator.cpp
+@@ -56,8 +56,7 @@ UserGestureIndicator::UserGestureIndicator(Optional<ProcessingUserGestureState>
+ 
+     if (state)
+         currentToken() = UserGestureToken::create(state.value(), gestureType);
+-
+-    if (document && currentToken()->processingUserGesture() && state) {
++    if (document && currentToken() && currentToken()->processingUserGesture() && state) {
```

Comment:
alternatively you could swap `state` with `currentToken()->processingUserGesture()`

Reply:
ah, that's much cleaner.